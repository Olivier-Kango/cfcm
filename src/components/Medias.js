import React from 'react';
import '../styling/Medias.scss';

const Medias = () => (
  <div className="medias">
    MÃ©dias
  </div>
);

export default Medias;
