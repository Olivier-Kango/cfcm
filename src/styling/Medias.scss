.medias {
  padding: 16% 5%;
}
