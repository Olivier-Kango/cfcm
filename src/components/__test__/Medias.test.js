import { render, screen } from '@testing-library/react';
import Medias from '../Medias';

describe('Testing the Medias page', () => {
  test('should render', () => {
    render(<Medias />);
    const about = screen.getByText(/MÃ©dias/i);
    expect(about).toMatchSnapshot();
  });
});
