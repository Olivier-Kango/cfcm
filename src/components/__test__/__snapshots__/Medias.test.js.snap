// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the  page should render 1`] = `
<div>
  Médias
</div>
`;

exports[`Testing the Contact page should render 1`] = `
<div>
  Contact
</div>
`;

exports[`Testing the Medias page should render 1`] = `
<div>
  Médias
</div>
`;
